client/js/gdpr/chart.js

```diff
@@ -114,8 +115,29 @@ export class GdprChart extends HTMLElement {
     this.clearHover();
   }
 
-  onScroll() {
-    this.clearHover();
+  onMouseWheel(ev) {
+    if (ev.ctrlKey) {
```

Comment:
so if the cltr key is pressed then we update the zoom, otherwise its cleared?

Reply:
yes